### ISSUE TITLE
Track down issue that causes consensus rounds to get held up

### ISSUE THREAD

**Issue by rkeene** [perlin-network/wavelet#171]
Track down issue that causes consensus rounds to get held up, currently experienced on Linux, and not Mac/Darwin

**Comment by AhmadMuzakkir** [comment]
I could not reproduce the issue on the linux machine.

Here's that I did.

1. Start 3 Nodes.
```
$ go run *.go -api.port 9000
$ go run *.go -port 3001 -api.port 9001 -wallet config/wallet2.txt 127.0.0.1:3000
$ go run *.go -port 3002 -api.port 9002 -wallet config/wallet3.txt 127.0.0.1:3000
```
2. On Node 1, spawn the `perl clicker` smart contract.
```
$ s contracts/perl_clicker.wasm
```
3. Run the benchmark with a payload:
```
$ go run *.go remote -payload f28ab8a9745fdc5170c03bba55051ad620ce3d9f6792b0b32a3161a742c8f2de000000000000000040420f0000000000000000000000000005000000636c69636b00000000
```

The rounds are being finalised.
The `accepted_tps` is around 240 - 270.

**Comment by a-urth** [comment]
We need either get more people to test it, or to ask @iwasaki-kenta to do more testing locally, since so far no one was able to reproduce it.

**Comment by AhmadMuzakkir** [comment]
I pushed an update to the `querying-issues` branch.
The update adds `-poll` flag. Using the flag, the benchmark will poll the consensus and for every `round_end` event, fetch the memory pages.
Any errors will be log.

Using the updated benchmark, I still could not reproduce the issue.
I ran the benchmark for about 40 minutes, the consensus still works, although slower (2 - 6 seconds) after around 30 minutes. The `accepted_tps` is around 80 - 100.
